### PR TITLE
Fix view and define profile view

### PR DESCRIPTION
**Pull request by wowKimsohyun17**
## Related Tickets:
- [#Ticket ID](https://edu-redmine.sun-asterisk.vn/issues/30257)

## WHAT this PR does?
profile view and home controller

## Notes Impacted Areas:
- N/A.

## Attachments:
Screenshot from 2020-10-26 15-21-08

### CODE REVIEW

**Comment by wowKimsohyun17** [comment]
ready

File: resources/views/layouts/profile_layout.blade.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+                        <span>Về trang chủ</span>
+                    </a>
+                </div>
+            </header>
+
+            @yield('content')
+
+        </section>
+    </body>
+</html>
```
Comment (huyenttm-0901) [inline resources/views/layouts/profile_layout.blade.php]: xem lại file này

File: resources/views/user/component/profile.blade.php [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+                    <div><i class="fas fa-chevron-right arrow-color"></i></div>
+                </a>
+                <a href="" class="user-detail-link">
+                    <section class="detail-item">
+                        <span>Mật khẩu</span>
+                        <span class="detail-description">Hahaha</span>
+                    </section>
+                    <div><i class="fas fa-chevron-right arrow-color"></i></div>
+                </a>
+            </section>
```
Comment (huyenttm-0901) [inline resources/views/user/component/profile.blade.php]: file này nữa

**Comment by wowKimsohyun17** [comment]
ready

File: resources/views/layouts/profile_layout.blade.php [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="{{ asset('plugins/fontawesome-free/css/all.min.css') }}">
+        <link rel="stylesheet" href="{{ asset('css/auth.css') }}">
+        <link rel="stylesheet" href="{{ asset('css/profile.css') }}">
+        <title>Login</title>
```
Comment (huyenttm-0901) [inline resources/views/layouts/profile_layout.blade.php]: i18n

**Comment by wowKimsohyun17** [comment]
ready

**Pull request merged**